### PR TITLE
Proxmoxer_vm_manager

### PR DESCRIPTION
**Pull request by KostiaLevchenko**
- requirements update
- added functionality get proxmox nodes
- added functionality get proxmox VM list
- added functionality delete VM in  proxmox 
- added functionality clone VM in proxmox
- added functionality get VM ip address in proxmox 
- added functionality start VM in proxmox

### CODE REVIEW

File: kubehub/proxmox/nodes_list.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from proxmoxer import ProxmoxAPI
+import json
+
+
+@csrf_exempt
+def nodes_list(request):
```
Comment (kopachevsky) [inline kubehub/proxmox/nodes_list.py:8]: node_list

File: kubehub/proxmox/vm_delete.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+from proxmoxer import ProxmoxAPI
+import json
+
+
+@csrf_exempt
+def vm_delete(request):
+    if request.method == 'POST':
+        try:
+            data = json.loads(request.body)
+            proxmox = ProxmoxAPI(host=data["proxmox_ip"], user='root@pam', password=data["password"], verify_ssl=False)
```
Comment (kopachevsky) [inline kubehub/proxmox/vm_delete.py:12]: expose it as function, don't send passwrod and ips via rest call

File: kubehub/proxmox/vm_list.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import json
+
+
+@csrf_exempt
+def vm_list(request):
+    if request.method == 'POST':
+        try:
+            data = json.loads(request.body)
+            proxmox = ProxmoxAPI(host=data["proxmox_ip"], user='root@pam', password=data["password"], verify_ssl=False)
+            vm = proxmox.cluster.resources.get(type='vm')
```
Comment (kopachevsky) [inline kubehub/proxmox/vm_list.py:13]: vms?

**Review by kopachevsky — APPROVED** [review]

File: kubehub/migrations/0002_auto_20200128_1204.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+    dependencies = [
+        ('kubehub', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='cloudprovider',
+            name='cp_type',
+            field=models.CharField(choices=[('PROX_MOX', 'PROX_MOX'), ('AWS', 'AWS'), ('GCP', 'GCP')], max_length=20),
```
Comment (kopachevsky) [inline kubehub/migrations/0002_auto_20200128_1204.py:16]: for public view change PROX_MOX to Proxmox?

Comment (KostiaLevchenko) [reply kubehub/migrations/0002_auto_20200128_1204.py:16]: fixed in https://github.com/dedicatted/kubehub-backend/pull/6       415f50a

**Pull request merged**